test/core/gc/array.wast

```diff
@@ -186,11 +193,13 @@
 
 (assert_return (invoke "new") (ref.array))
 (assert_return (invoke "new") (ref.eq))
-(assert_return (invoke "get" (i32.const 0)) (i32.const 1))
+(assert_return (invoke "get_u" (i32.const 0)) (i32.const 1))
+(assert_return (invoke "get_s" (i32.const 2)) (i32.const -1))
```

Comment:
It would be good to call each of `get_u` and `get_s` with both indices.

Reply:
Done.